frontend/packages/kubevirt-plugin/integration-tests/tests/kubevirt.login.scenario.ts

```diff
@@ -0,0 +1,18 @@
+import { execSync } from 'child_process';
+import { browser } from 'protractor';
+import { appHost } from '../../../../integration-tests/protractor.conf';
```

Comment:
Please avoid relative paths that point outside the monorepo. Let's do two things here:

1, add `integration-tests/package.json`

```json
{
  "name": "@console/integration-tests",
  "version": "0.0.0-fixed",
  "description": "Console integration tests to be moved into appropriate packages",
  "private": true
}
```

2, update root `package.json` & run `yarn` to update symlinks in `node_modules`

```json
"workspaces": [
  "packages/*",
  "public",
  "integration-tests"
],
```

3, update all imports in plugin packages

```diff
-import { appHost } from '../../../../integration-tests/protractor.conf';
+import { appHost } from '@console/integration-tests/protractor.conf';
```

Reply:
we do it in https://github.com/openshift/console/pull/2051 , see comment about links.

Reply:
> Can we use @console/internal-integration-tests ?

Ref: https://github.com/openshift/console/pull/2051/files#r313030053 

p.s.
if changing here, this should be changed in all relevant places.

Reply:
Done. Told me about this comment: https://github.com/openshift/console/pull/2051#discussion_r313030053
Is it relevant for this PR? Should I try to incorporated it?

Reply:
@vojtechszocs So I added this change and then removed it to not create conflict with Kobi's PR #2051

Reply:
OK, we can do it in Kobi's PR #2051.